diag_manager/diag_output.F90

```diff
@@ -383,9 +386,11 @@ SUBROUTINE write_axis_meta_data(file_unit, axes, fileob, time_ops, time_axis_reg
                          call write_data(fptr, axis_name, axis_data(istart:iend) )
                       endif
                     type is (FmsNetcdfFile_t) !< For regional X and Y axes, treat as any other axis
-                         call register_axis(fptr, axis_name, dimension_length=size(axis_data))
-                         istart = lbound(axis_data,1)
-                         iend = ubound(axis_data,1)
+                         call mpp_get_global_domain(domain, gstart, gend, glength)  !< Get the global indicies
+                         call mpp_get_compute_domain(domain, cstart, cend, clength) !< Get the compute indicies
```

Comment:
I believe these interfaces have a lot of optional argument references.  You should refer to the optional arguments with XXX=YYY.

The variable "gend" was already defined for use with a "gbegin" pair.  Are you using "gend" here safely?

Reply:
It should be able to be used here because it is redefined where it's used later.  I guess I didn't realize it, though.  @ngs333 should probable change the name of these variables.